### DISCUSSION TITLE
Can't copy "share" icon from documentation

### DISCUSSION THREAD

**Discussion by itedya** [tailwindlabs/heroicons#1123] [Help] [answered]
In documentation, "Copy SVG" and "Copy JSX" buttons do not appear when mouse is over the "share" icon.
I'm using firefox 122.0

**Comment by adamwathan** [comment, marked as answer]
Can't reproduce here, any chance it's related to a browser extension or something that's doing something funny to "Share"-style links/buttons and catching this by mistake?

https://github.com/tailwindlabs/heroicons/assets/4323180/c8bbb2b1-43f0-484d-8e88-71c73dd518d6

**Reply by itedya** [reply]
I completely forgot about extensions, that fixes the issue, thanks.
I have a very agressive uBlock setup and I have already detected the buggy filter list, will report it to the correct repo.
